### DISCUSSION TITLE
Is this geo-AR still maintained ?

### DISCUSSION THREAD

**Discussion by jeromeDms** [CariusLars/ar_flutter_plugin#186] [General]
Hi all
I found this geolocation based AR plugin, I was wondering if this package is still maintained ?
Thanks

**Comment by hlefe** [comment]
Hello, you can use my fork which is maintained with the latest APIs : 
https://github.com/hlefe/ar_flutter_plugin_flutterflow
https://pub.dev/packages/ar_flutter_plugin_flutterflow
